src/ForkTsCheckerWebpackPluginConfiguration.ts

```diff
@@ -19,6 +19,7 @@ interface ForkTsCheckerWebpackPluginConfiguration {
   issue: IssueConfiguration;
   formatter: FormatterConfiguration;
   logger: LoggerConfiguration;
+  enableOverlay: boolean;
```

Comment:
Generally it looks ok, but I would put it as a part of the logger configuration - we could have `logger.devServer: boolean` option. I would not use `overlay` name because it's about reporting to the webpack-dev-server in general - the overlay is an option in webpack-dev-server :)

Reply:
:+1: